contracts/system-contracts/hedera-account-service/README.md

```diff
@@ -0,0 +1,11 @@
+## Hedera Account Service (HAS) System Contract Functions
+
+The Hedera Account Service (HAS) System Contract is accessible at address `0x16a` on the Hedera network. This contract introduces new account proxy contract for HBAR allowances, enables querying and granting approval of HBAR to a spender account from within smart contracts, allows developers to grant, retrieve, and manage HBAR allowances directly within smart contract code.
```

Comment:
Suggest something like - This contract introduces a new account proxy contract in order to interact with a contract for such functionality as HBAR allowances etc..

Reply:
Updated.